pkg/config/config.go

```diff
@@ -36,6 +37,7 @@ func NewFromFlags() (*Config, error) {
 	flag.BoolVarP(&config.Cluster, "cluster", "c", true, "enable Cluster collector")
 	flag.StringVarP(&config.Context, "context", "x", "", "kubeconfig context")
 	flag.BoolVarP(&config.ExitError, "exit-error", "e", false, "exit with non-zero code when issues are found")
+	flag.BoolVarP(&config.KubentVersion, "version", "v", false, "fetch version information of kubent binary")
```

Comment:
This doesn't match the output added to the README. I would prefer that formulation:
```
prints the version of kubent and exits
```

Reply:
Thats actually what I had first. Done!